scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/soic.yaml

```diff
@@ -646,3 +646,30 @@ SOIC-28W_7.5x18.7mm_P1.27mm:
   pitch: 1.27
   num_pins_x: 0
   num_pins_y: 14
+  
+Infineon_PG-DSO-8-59:
+  custom_name_format: 'Infineon_PG-DSO-8-59' 
+  size_source: 'https://www.infineon.com/cms/en/product/packages/PG-DSO/PG-DSO-8-59'
+  body_size_x: 
```

Comment:
You can try this:
  body_size_x: 
    maximum: 7.6
    nominal: 7.6
    minimum: 7.4